docs/guide/tutorials/connect/connect.txt

```diff
@@ -1,3 +1,95 @@
 ==================
 Connect to MongoDB
 ==================
+
+Use the ``client.connect`` method to connect to a running MongoDB deployment.
+
+Connect to a Single MongoDB Instance
+------------------------------------
+
+To connect to a single MongoDB instance, specify the URI of the MongoDB
+instance to connect to.
+
+In the following example, the
+:manual:`URI connection string </reference/connection-string/>`
+specifies connecting to a MongoDB instance that is running on
+``localhost`` using port ``27017``. The ``myproject`` indicates the database
+to use.
+
+.. literalinclude:: /includes/basic-connection.js
+   :language: js
+
+For more information on the URI connection string, see
+:manual:`URI connection string </reference/connection-string/>` .
+
+Connect to a Replica Set
+------------------------
+
+To connect to a :manual:`replica set </core/replication-introduction/>` ,
+include a seedlist of replica set members and the name of the replica set in the
+:manual:`URI connection string </reference/connection-string/>` .
+
+In the following example, the connection string specifies two of the replica set members running on ``localhost:27017`` and ``localhost:27018`` and the name of the replica set (\ ``foo``\ ). 
+
+.. literalinclude:: /includes/connect-to-replicaset.js
+   :language: js
+
+For more information on the URI connection string, see
+:manual:`URI connection string </reference/connection-string/>` .
+
+Connect to Sharded Cluster
+--------------------------
+
+To connect to a :manual:`sharded cluster </core/sharded-cluster-components/>` , specify the ``mongos`` instance or instances in the :manual:`URI connection string </reference/connection-string/>` .
+
+In the following example, the connection string specifies the ``mongos`` instances running on ``localhost:50000`` and ``localhost:50001``.
+
+.. literalinclude:: /includes/connect-to-sharded-cluster.js
+   :language: js
+
+For more information on the URI connection string, see
+:manual:`URI connection string </reference/connection-string/>` .
+
+Connection Options
+------------------
+
+You can specify various connection settings in the :manual:`URI connection string </reference/connection-string/>` .
+
+For example, you can specify TLS/SSL and authentication setting.
+
+.. code-block:: js
+
+   const { MongoClient } = require('mongodb');
+   const assert = require('assert');
+   const fs = require('fs');
+
+     // Read the certificate authority
+   const ca = [fs.readFileSync(__dirname + "/ssl/ca.pem")];
+   const cert = fs.readFileSync(__dirname + "/ssl/client.pem");
+
+   // Connection URL
+   const url = 'mongodb://dave:password@localhost:27017?authMechanism=DEFAULT&authSource=db&ssl=true"';
+
+   // Create a client, passing in additional options
+   const client = new MongoClient(url,  {
+     sslValidate: true,
+     sslCA: ca,
+     sslCert: cert
+   });
+
+   // Use connect method to connect to the Server
+   client.connect().then(async function() {
```

Comment:
switch to es6 arrow fn
`   client.connect().then(async function() {` should be `client.connect().then(async () => {`